### ISSUE TITLE
Pretrained hife model

### ISSUE THREAD

**Issue by jmaba** [Gina-YUE/HiFE#3]
Could you please upload also the pth (one of them) for the hife model? Maybe a GDrive where to upload that pth model

**Comment by twilight2001** [comment]
yes also interested in checking it out, we're non-profit government research writing deepfake detection consultation proposal, thanks  !
